src/components/HomepageNavbar/index.tsx

```diff
@@ -5,15 +5,15 @@ import Wrapper from '@site/src/components/Wrapper';
 const navLinks = [
   {
     label: 'Connectors',
-    href: '/docs/connectors/connector-list',
+    href: '/docs/using/connectors/list',
   },
   {
     label: 'Documentation',
     href: '/docs',
   },
   {
-    label: 'Changelog',
-    href: '/changelog',
+    label: 'Conduit Platform',
```

Comment:
Nitpick: Conduit Platform or The Conduit Platform? I'm not sure myself.:)

Reply:
The product name is `Conduit Platform`, but it could be expressed as "The Conduit Platform" in other context. See https://meroxa.com/